### PR TITLE
Relates to #265: Do not require consistent snapshots.

### PR DESCRIPTION
**Pull request by ergonlogic**

### CODE REVIEW

**Comment by ergonlogic** [comment]
This appears to force 'consistent snapshot' to be true. When we test this with `"consistent_snapshot": false,` in `1.root.json`, we get:

```
> pre-file-download: Tuf\ComposerIntegration\Plugin->preFileDownload
                                                                                       
  [Tuf\Exception\MetadataException]                                                    
  Object(ArrayObject)[signed][consistent_snapshot]:                                    
      This value should be equal to true. (code 478618a7-95ba-473d-9101-cabd45e49115)
```

File: src/Client/Updater.php [hunk 1]
```diff
@@ -103,7 +103,13 @@ class Updater
      */
     protected $universalVerifier;
 
-    /**
+  /**
+   * The root metadata.
+   * @var \Tuf\Metadata\MetadataBase|null
+   */
+    private $rootMetadata;
```
Comment (phenaproxima) [inline src/Client/Updater.php:110]: Do we really need to keep the root metadata as a class property? I feel like this is a way to accidentally introduce side effects later.

**Review by phenaproxima — CHANGES_REQUESTED** [review]
I think that there are generally two things missing here: a generous sprinkling of comments, and test coverage.

**Comment by phenaproxima** [comment]
Superseded by #278.

**Pull request closed without merging**